### DISCUSSION TITLE
I can't type with my os on an actual computer

### DISCUSSION THREAD

**Discussion by Meomeowbowow** [CosmosOS/Cosmos#2543] [Q&A]
I have made an OS in Cosmos and I booted onto a laptop and i cant type with. This happens with any computer i use!
Does anyone know what is happening?

**Comment by ghost** [comment]
I think you should use a PS/2 Keyboard

**Reply by Meomeowbowow** [reply]
i dont have a ps/2 keyboard

**Reply by AsertCreator** [reply]
cosmos doesnt support usb yet, so you cant use usb keyboard with it.
